src/components/CaseStudies/CaseStudyLayout.tsx

```diff
@@ -2,7 +2,7 @@ import { FunctionComponent } from 'react'
 
 import { kebabCase } from 'lodash'
 
-import { ContentSection, RequestDemoForm, MediaQuote } from '@components'
```

Comment:
`MediaQuote` was only being used in the old case study layout. All case studies will soon change to the new layout. So this component will be deprecated soon anyway.